demos/m5/README.md

```diff
@@ -2,9 +2,19 @@
 
 This directory contains the resources needed for running the release demo for Milestone 5.
 
-## Environments
+# Bootstraping
 
-This demo can run in two different environments:
+After bootstrapping the cluster with App Studio, you can prepare the demo by running the [boostrap.sh](demos/m5/boostrap.sh) script.
```

Comment:
script should be named bootstrap.sh with two t's

Reply:
I swear I don’t do it on purpose!

Reply:
pastebin catches it immediately.. try that instead of vscode